packages/components/src/components/text-field/text-field.tsx

```diff
@@ -72,6 +72,8 @@ export class TextField {
   @Prop() inputId?: string;
   /** (optional) input background transparent */
   @Prop() transparent?: boolean;
+  /** (optional) input list */
+  @Prop() list?: string | Array<Record<'value' | 'displayValue', string>>;
```

Comment:
I wouldn't take data in the `list` attribute, but keep it simple as take the id of the `<datalist>` element with should be linked with.

That means we can skip the `<datalist>` element and all the `readData` implementation.